docs/contributing/CONTRIBUTING.md

```diff
@@ -83,8 +87,7 @@ Go ahead and edit anything.
 ## Making Pull Requests
 
 The preferred and easiest way to contribute changes to Matrix is to fork the
```

Comment:
```suggestion
The preferred and easiest way to contribute changes to Matrix is to fork the
```

To matrix????

Reply:
Open an issue, this is text from Mjolnir.